civis/tests/test_io.py

```diff
@@ -186,7 +186,7 @@ def test_civis_file_to_table_table_exists(m_run_cleaning,
     mock_civis.get_database_id.return_value = 42
     mock_civis.default_credential = 713
 
-    mock_civis.get_table_id.return_value = 42
+    mock_civis.databases.get_schemas_tables.return_value = 42
```

Comment:
It probably wouldn't affect this test, but for completeness, since `client.databases.get_schemas_tables` is supposed to return a `civis.response.Response` object, maybe the return value here should match that?